projects/dynamicforms/src/components/currency/currency.component.html

```diff
@@ -0,0 +1,30 @@
+<ng-container *ngIf="group">
+  <div class="dynamic-field" [formGroup]="group" [hidden]="config.hidden">
+    <div class="form-group">
+      <label class="col-form-label">
+        {{ label }}
+        <ng-container *ngIf="!config.required">
+          {{ 'dynamicforms.optional' | cxTranslate }}
+        </ng-container>
+      </label>
+      <div class="currency d-flex">
+        <span
+          class="currencyCode d-flex justify-content-center align-items-center"
+          >{{ currentCurrency$ | async }}</span
+        >
+        <input
+          class="form-control"
+          type="text"
```

Comment:
`type=number`? User shouldn't be able to enter anything other than numbers here.

Reply:
I don't know if this looks good to you?

![Screenshot 2021-08-25 at 11 37 49](https://user-images.githubusercontent.com/75082466/130767378-b2da6106-e0ab-4544-aa25-0747401e96fc.png)

Validation anyway comes from the JSON form definition.

Reply:
I don't see need for these arrows at the end of the input field...

Reply:
As we agreed I'll use type=text.